src/content/blog/astro-460.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: "Astro 4.6"
+description: "Astro 4.6 is here! This release includes a new manual routing strategy for internationalization, the ability to move the dev toolbar, experimental support for CSRF protection and more."
+publishDate: "April 11, 2024"
+authors:
+  - erika
+  - ema
+  - matthew
+  - nate
+  - bjorn
+coverImage: "/src/content/blog/_images/astro-460/post-header-4.6.webp"
+socialImage: "/src/content/blog/_images/astro-460/og-image-4.6.webp"
+lang: "en"
+---
+
+import BlogSummary from "/src/components/BlogSummary.astro";
+
+<BlogSummary>
+  After a short break post-launch week, Astro 4.6 is now here and we're back to our regular schedule! This release includes a new manual routing strategy for internationalization, experimental support for CSRF protection, a new feature for the dev toolbar and more.
+</BlogSummary>
+
+Full release highlights include:
+
+- [**Manual routing strategy for internationalization**](#manual-routing-strategy-for-internationalization)
+- [**Ability to move the dev toolbar**](#ability-to-move-the-dev-toolbar)
+- [**Experimental: Support for CSRF protection**](#experimental-support-for-csrf-protection)
+- [**Cookies improvements**](#cookies-improvements)
+
+To upgrade an existing project, use the automated `@astrojs/upgrade` CLI tool. Alternatively, upgrade manually by running the upgrade command for your package manager:
+
+```sh
+# Recommended:
+npx @astrojs/upgrade
+
+# Manual:
+npm install astro@latest
+pnpm upgrade astro --latest
+yarn upgrade astro --latest
+```
+
+## Deprecated support for older versions of Node
+
+In accordance with [our Node support and upgrade policy](https://docs.astro.build/en/upgrade-astro/#nodejs-support-and-upgrade-policies), this version of Astro deprecate support for versions of Node lower than 18.17.1, Node 19 and versions of Node lower than 20.3.0. As always, this decision wasn't reached lightly, and we believe this change will result in a more stable and secure Astro experience for all users.
+
+Astro will still work on these older versions of Node until Astro 5, however you may see a warning when installing Astro and certain features may not work as expected.
+
+## Manual routing strategy for internationalization
+
+Astro 4.6 introduces a new manual routing strategy for internationalization. This new strategy allows you to take total control over the routing of your internationalized Astro website, for cases where the default routing strategy doesn't quite meet your needs.
+
```

Comment:
I would add a short paragraph, 2 or 3 sentences, explaining how you can use the same helper functions that the other routing strategies use internally, and that you can even use the core middleware, but place it after your own as a way to circumvent the default behaviors.

Reply:
(You can see the newly updated changeset for an example/description here)